From a8a39f3210a644e04dbb1eaefe484578a7913ca8 Mon Sep 17 00:00:00 2001
From: AsavarTzeth <asavartzeth@gmail.com>
Date: Tue, 12 Jun 2018 20:44:19 +0200
Subject: [PATCH 1/2] Add OARS 1.0 data to AppStream file

The "Open Age Ratings Service" free and commercial software to
inform users about data collection, and content that may be violent or
otherwise should be age restricted.

Basically it is just a way to tag content in so that users (and parents)
can be informed prior to installation and potential exposure.

While not that relevant to PulseEffects, given all values are set to
"none", it is still valuable to show this. That and it is now listed as
a submission requirement at Flathub.
---
 ...om.github.wwmm.pulseeffects.appdata.xml.in | 22 +++++++++++++++++++
 1 file changed, 22 insertions(+)

diff --git a/data/com.github.wwmm.pulseeffects.appdata.xml.in b/data/com.github.wwmm.pulseeffects.appdata.xml.in
index abe58b7..219ce8d 100644
--- a/data/com.github.wwmm.pulseeffects.appdata.xml.in
+++ b/data/com.github.wwmm.pulseeffects.appdata.xml.in
@@ -56,6 +56,28 @@
       <caption>Calibration</caption>
     </screenshot>
   </screenshots>
+  <content_rating type="oars-1.0">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">none</content_attribute>
+    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-audio">none</content_attribute>
+    <content_attribute id="social-location">none</content_attribute>
+    <content_attribute id="social-contacts">none</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
+    <content_attribute id="money-gambling">none</content_attribute>
+  </content_rating>
   <update_contact>contact_AT_hembas.se</update_contact>
   <translation type="gettext">PulseEffects</translation>
 </component>
-- 
2.18.0

